2020-11-25 05:06:23,355: DEBUG: NEW RUN: Client, TEST: Write_Multiple_Clients_Five_Threads5
2020-11-25 05:06:23,355: DEBUG: The client is starting.
2020-11-25 05:06:23,355: DEBUG: The client is starting on port 22726
2020-11-25 05:06:23,355: DEBUG: Sent WRQ: b'\x00\x02shutdown.txt\x00netascii\x00'
2020-11-25 05:06:23,357: DEBUG: File exists, can send it
2020-11-25 05:06:23,357: DEBUG: First packet received. Raw bytes: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 05:06:23,358: DEBUG: New packet from: ('127.0.0.1', 61512), contents: b'\x00\x05\x00\x02filename already exists at destination\x00'
2020-11-25 05:06:23,358: DEBUG: Sending: shutdown.txt
2020-11-25 05:06:23,358: ERROR: Shutting down due to error packet. Error code: 2, error message: b'filename already exists at destination'
2020-11-25 05:06:23,358: WARNING: File didnt close on its own, closing now
2020-11-25 05:06:23,458: DEBUG: Client finished.
2020-11-25 05:06:23,458: DEBUG: Closing Socket
